Adyen/Integration/02_ManageAdyenConnection.md

```diff
@@ -0,0 +1,161 @@
+
+For detailed information, see [Edit PSP connection](../../Payments/Integration/01_ManageConnection.md#edit-psp-connection) in the *Payments* documentation.
+
+
+
+## Configure Adyen connection
+
+Configure the Adyen&trade; connection after you have created it. In the *Payments* connection settings, you can enter the event codes that are to be ignored. Alternatively, you can select the unwanted event codes in the Adyen&trade; webhook settings. 
+
+The Adyen&trade; API has two endpoints, one for a synchronous and one for an asynchronous processing. If you decide for asynchronous processing, what is recommended, you can configure the number of workers to process messages in asynchronous processing. For detailed information, see [Define synchronous or asynchronous processing](./01_ManageAdyenSettings.md#define-synchronous-or-asynchronous-processing).
+
+
+#### Prerequisites
+
+In the *Actindo Core1 Platform*, you are permitted to create connections in *Payments*.
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+ 
+ ![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+ 1. Click the Adyen&trade; connection you want to edit in the list of connections.   
+   The *Edit connection* view is displayed. By default, the *Credentials* tab is selected.
+
+    ![Edit Adyen settings](../../Assets/Screenshots/Adyen/Integration/EditCredentials.png "[Edit Adyen credentials]")
+
+2. Click the *Settings* tab.   
+  The *Ignored event codes* setting is displayed by default.
+
+   ![Ignored event codes](../../Assets/Screenshots/Adyen/Integration/IgnoredEventCode.png "[Ignored event codes]")
+
+3. If desired, enter an event code to be ignored in the *Key* field and in the *Value* field as well. Note that you can also define the event codes to be ignored in the Adyen&trade; backend. For detailed information, see [Create webhook in Adyen](./01_ManageAdyenSettings.md#create-adyen-webhook).
+    The event code to be ignored is entered. 
+
+    ![Ignored event code](../../Assets/Screenshots/Adyen/Integration/IgnoredEventCodeDone.png "[Ignored event code]")
+
+4. If desired, enter further event codes to be ignored. To add a new row, click the ![Add](../../Assets/Icons/Plus04.png "[Add]") (Add) button. 
+
+5. Click the *Number of workers handling messages* setting.   
+   The *Number of workers handling messages* setting is displayed. 
+   ![Number of workers](../../Assets/Screenshots/Adyen/Integration/NumberOfWorkers.png "[Number of workers]")
+
+6. Enter the number of workers you want to use for asynchronous processing of messages. For connections with a lot of traffic, a number between 5-10 might be sufficient. See also [Define synchronous or asynchronous processing](01_ManageAdyenSettings.md#define-synchronous-or-asynchronous-processing).
+
+7. If desired, you can apply the standard settings by enabling the ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") (Apply default) toggle in the top right corner of the tab.
```

Comment:
```suggestion
7. If desired, you can apply the standard settings by enabling the ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") (Apply from default) toggle in the top right corner of the tab.
```
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/f2cc7619-87cc-4f8d-98c4-4e7eb3d9947e)

Apply from default ist der Name des Toggles, daher sollte es in kursiv, finde ich. Oben Beispiel aus Omni-Channel.